### PR TITLE
Fixed show bench button in web UI

### PR DESCRIPTION
**Pull request by davydovanton**
Moved show button on alert-warning line. Fixed displaying for mobile
screen.

##### Desktop
screenshot 2015-04-26 02 28 37

###### Mobile
screenshot 2015-04-26 02 28 09
screenshot 2015-04-26 02 28 25

### CODE REVIEW

File: app/assets/stylesheets/modules/_body.scss [hunk 1]
```diff
@@ -2,3 +2,36 @@
...
+  #chart-container {
+    #benchmark-types-form-show {
+      float: left;
+    }
+
+    .alert-warning-moved-right {
+      margin-left: 245px;
+    }
+  }
+}
```
Comment (tgxworld) [inline app/assets/stylesheets/modules/_body.scss]: Lets move all of this into `app/assets/stylesheets/modules/_chart-container.scss`.

Comment (tgxworld) [inline app/assets/stylesheets/modules/_body.scss]: I prefer to use `em` instead of `px` :smile:

File: app/assets/stylesheets/modules/_body.scss [hunk 2]
```diff
@@ -2,3 +2,36 @@
...
+    padding-bottom: 8px;
+    padding-top: 8px;
+  }
+
+  .alert-warning {
+    padding: 8px;
+  }
+}
+
+@media (max-width: 547px) {
```
Comment (tgxworld) [inline app/assets/stylesheets/modules/_body.scss]: Could we use foundation's media queries here instead? http://foundation.zurb.com/docs/media-queries.html :smile_cat:

File: app/assets/stylesheets/modules/_body.scss [hunk 3]
```diff
@@ -2,3 +2,36 @@
...
+}
+
+@media (min-width: 548px) {
+  #chart-container {
+    #benchmark-types-form-show {
+      float: left;
+    }
+
+    .alert-warning-moved-right {
+      margin-left: 245px;
```
Comment (tgxworld) [inline app/assets/stylesheets/modules/_body.scss]: hmm hacking in the margin here doesn't seem right to me. Could you see if there is another way to do it by setting the `columns` it takes up? Thanks!!

**Comment by davydovanton** [comment]
@tgxworld fixed all, I'm sorry it took so long :sweat:
*P.S.: also I update screenshots in pull request description*

**Comment by tgxworld** [comment]
@davydovanton one of the test failed :sweat_smile:

**Comment by tgxworld** [comment]
@davydovanton Thank you for the work but with the redesign this PR is no longer valid. Sorry about that! Should have made it known that we were planning a redesign.

**Comment by davydovanton** [comment]
@tgxworld no problem, it's my fault, I dragged PR. Thanks :green_heart:

**Comment by tgxworld** [comment]
@davydovanto no worries. It is no ones fault :) thank you so much for contributing!

**Pull request closed without merging**